tests/e2e/utils.go

```diff
@@ -126,3 +126,24 @@ func matchResponseSubString(url, match string, retry, sleep int) bool {
 	fmt.Printf("Could not get the match string \"%s\" in %d seconds\n", match, i)
 	return false
 }
+
+// Creates a directory in the path specified. It takes one argument
+// dirNameFullpath (string) - Directory name along with the path
+func createDir(dirNameFullpath string) error {
+	return os.MkdirAll(dirNameFullpath, 0777)
+}
+
+// Creates a file in a specified path. It takes two argument
+// filePath (string) - The path where you want to create the file
+// fileName (string) - Name of the file
+func createFileAtPath(filePath string, fileName string) error {
+	_, err := os.Stat(filePath + "/" + fileName)
```

Comment:
instead of concatenating strings with `/` you should use `filepath.Join`, this will handle path properly on different platforms like windows

Reply:
Thanks for the comment. I will consider the suggestion for sure, however i am getting a panic out message when trying locally 
```
[...]
panic: 
Your test failed.
Ginkgo panics to prevent subsequent assertions from running.
Normally Ginkgo rescues this panic so you shouldn't see it.
[...]
```

Reply:
this is realted to `filepath.Join`?

Reply:
Have you checked https://onsi.github.io/ginkgo/#marking-specs-as-failed ?